### PR TITLE
Added Steps for checking if node and npm is installed.

### LINKED ISSUE #245: Enhancement in Readme.
**Is your feature request related to a problem? Please describe.**
Currently, in the second point of Getting Started, It does not provide the option for to user to check if he/she already has npm or node
Also, it does not show options to install it on windows which might confuse some users or contributors if it could be run in windows or not.

**Describe the solution you'd like**
- Providing the options for checking if node and npm are installed.
- Guiding user to install node and npm in windows.

**Describe alternatives you've considered**
Solutions described above.

**Additional context**
This might not seem to be a big issue but will certainly help new contributors or users a lot.

@vojtechjelinek  Sir, Please let me know if you think this might prove helpful. If yes. Please do assign me . Thanks.

### PR DESCRIPTION
**Pull request by Naman-1234**
## Explanation

Fixes #245 
## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by Naman-1234** [comment]
@vojtechjelinek  and @jameesjohn  Sir I think work in this PR is done. Please review this. Thanks.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ Please refer to the following instructions to setup Oppiabot for the first time
 
 2. Install [Node.Js](https://nodejs.org/) and [NPM](https://www.npmjs.com/) on your machine using the following commands:
 
+    First check if you already have Node.js and NPM using:
+    ```bash
+    node -v
+    npm -v
+    ```
+    If these commands show some versions, then you can jump to Point No. 3, else proceed in this point:
```
Comment (vojtechjelinek) [inline README.md:31]:
```suggestion
    If these commands show some versions, then you can jump to Point No. 3, else proceed with these commands:
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks, one comment.

File: README.md (hunk 1 above)
Comment (Naman-1234) [reply README.md:31]: Sure sir, Changing them right away.

**Review by jameesjohn — APPROVED** [review]
LGTM!

**Pull request merged**